.github/workflows/devshell.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       - name: make check
         id: make_check
         working-directory: ./build
-        run: make V=1 check || (cat test-suite.log && false)
+        run: make V=1 check || (mkdir -p ${COREFILES_DIR} && find . -executable -a -type f | tar -cf ${COREFILES_DIR}/test-binaries.tar  --files-from=- && cat test-suite.log && false)
```

Comment:
I think saving the whole build directory could be better.

Reply:
You might be right, but it wasn't missed before and would produce a much larger tarball.

The core files produced by test executables are unusable, with this we stand a slight chance :) but feel free to improve it further.